server/cluster.go

```diff
@@ -387,6 +397,20 @@ func (c *RaftCluster) putStore(store *metapb.Store) error {
 	return nil
 }
 
+// RemoveStore marks a store as tombstone in cluster.
+func (c *RaftCluster) RemoveStore(storeID uint64) error {
+	store, _, err := c.GetStore(storeID)
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	// TODO: Gracefully remove store.
+	store.State = metapb.StoreState_Tombstone
+	store.Address = ""
```

Comment:
should keep the address, so we can see whether stores are tombstone. 
of course, https://github.com/pingcap/pd/pull/297 should update